### PR TITLE
Update data.json file and added my  details

### PR DESCRIPTION
**Pull request by Surajbokde**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -490,5 +490,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/60464414/104895387-d0a1ce00-599b-11eb-8bc0-58d79507051b.jpg",
     "issueNumber": 249
-  }
+  },
+  {
+    "name": "Suraj Bokde",
+    "githubUsername": "Surajbokde",
+    "imageUrl": "https://user-images.githubusercontent.com/66197682/105678184-d7d45900-5f12-11eb-9899-6e2757aa3a5f.png",
+    "issueNumber": 279
+}
```
Comment (AllanRegush) [inline data.json:499]: I think this bracket should be lined up like the rest.

**Review by AllanRegush — COMMENTED** [review]
Thank you for your pull request :+1: Just one small adjustment

**Comment by Surajbokde** [comment]
@AllanRegush check now ,also i change my ImageURL

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1:  Thank you

**Pull request merged**